scripts/create_AR_DeployInst.py

```diff
@@ -299,7 +299,8 @@ def update_power_demand(
         prototype
     '''
     power_gap[index:index + reactor_prototypes[prototype]
-              [1]] -= reactor_prototypes[prototype][0] * num_rxs
+            [1]] = power_gap[index:index+reactor_prototypes[prototype]
+            [1]] - reactor_prototypes[prototype][0] * num_rxs
```

Comment:
```suggestion
            [1]] -= reactor_prototypes[prototype][0] * num_rxs
```

Really, you should replace lines 301:304 with the following

```py
reactors = reactor_prototypes[prototype]
power_gap[index:index + reactors[1]] -= reactors[0] * num_rxs
power -= reactors[0] * num_rxs
```

reactor_prototypes[prototype] no longer gets repeated, and since you're modifying a value in place you can use `-=` to decrement by the specified value.

Reply:
Did you change the ‘reactor_types[prototype]’ situtation?

Reply:
Did I not? I thought I made the change you requested.

Reply:
I must have missed this one. The change has been made now.